docs/vllm_routing_replay_integration.md

```
@@ -0,0 +1,81 @@
+## API
+
+### `routing_replay_out` Parameter
+
+Available on:
+- `trtllm_fp8_block_scale_moe()`
+- `trtllm_bf16_moe()`
+- `trtllm_bf16_routed_moe()`
+- `fused_topk_deepseek()`
+
+**Spec:**
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)